src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import { useState } from "react";
 import "./App.scss";
+import Header from "./components/Header/Header";
```

Comment:
Legal, um dos benefícios para gente ter um `index.ts` dentro do folder do component seria o `readability` que seria o quão fácil é ler o seu código. 

Nesse caso o seu import ele está sendo redundante, agente podia melhorar criando um `index.ts` dentro do component, o objetivo seria simplificar o seu import para algo como o exemplo abaixo:

import Header from "./components/Header";

Dê uma olhadinha no `./components/ComponentExample/index.ts`

Reply:
Ok, i read the comments in the example index and understood, thank you very much. Already made the changes in the files.